src/Path/AccountSettings.jsx

```diff
@@ -45,8 +42,10 @@ const AccountSettings = () => {
 				email,
 				phoneNumber,
 			}));
+		}).catch((error) => {
+			console.error('Error fetching profile:', error);
 		});
-	}, []);
+	}, [isAuthenticated]);
```

Comment:
Why are we adding this everywhere

Reply:
whenever the isAuthenticated status changes, the login state needs to change to logout